### PR TITLE
Add status archived

### PR DESCRIPTION
**Pull request by MM3103**
Add status archived

### CODE REVIEW

File: src/main/java/my/pr/SpringBootStarter.java [hunk 1]
```diff
@@ -3,8 +3,10 @@
 import io.swagger.v3.oas.annotations.info.Info;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
+@EnableScheduling
```
Comment (AntonPozdeev) [inline src/main/java/my/pr/SpringBootStarter.java:9]: should be placed in particular place

File: src/main/java/my/pr/model/Order.java [hunk 2]
```diff
@@ -44,6 +45,13 @@ public Order() {
 
     @Column(name = "order_status")
     private Status orderStatus;
+
+    @Column(name = "create_time")
+    private OffsetDateTime create;
```
Comment (AntonPozdeev) [inline src/main/java/my/pr/model/Order.java:50]: lets make type as Date java.util.Date

File: src/main/java/my/pr/model/Order.java [hunk 3]
```diff
@@ -44,6 +45,13 @@ public Order() {
 
     @Column(name = "order_status")
     private Status orderStatus;
+
+    @Column(name = "create_time")
```
Comment (AntonPozdeev) [inline src/main/java/my/pr/model/Order.java:49]: "create_time" - creation_time or created_when

File: src/main/java/my/pr/model/Order.java [hunk 4]
```diff
@@ -44,6 +45,13 @@ public Order() {
...
+    private OffsetDateTime create;
+
+    @Column(name = "handling_time")
```
Comment (AntonPozdeev) [inline src/main/java/my/pr/model/Order.java:52]: handling_time -> MODIFIED_WHEN or modification_time

File: src/main/java/my/pr/repository/OrderRepository.java [hunk 5]
```diff
@@ -13,4 +13,7 @@ public interface OrderRepository extends JpaRepository<Order, UUID> {
 
     @Query(value = "select * from orders where email = :email", nativeQuery = true)
     List<Order> findByEmail(@Param("email") String email);
+
+    @Query(value = "select * from orders where (order_status = '1' or order_status = '2') and ((now() - handling_time)> '24 hours' )", nativeQuery = true)
```
Comment (AntonPozdeev) [inline src/main/java/my/pr/repository/OrderRepository.java:17]: order_status in ('1','2')

File: src/main/java/my/pr/service/OrderService.java [hunk 6]
```diff
@@ -112,4 +117,18 @@ private void emailMessage(Order savedOrder, AccessToken token) throws MessagingE
...
+        List<Order> orders = repository.findForArchived();
+        for (int i = 0; i < orders.size(); i++) {
+            Order newOrder = orders.get(i);
+            newOrder.setOrderStatus(Status.ARCHIVED);
+            repository.save(newOrder);
+        }
+    }
+
+    @Scheduled(fixedRateString = "3600000")
+    private void archivedStatus() throws InterruptedException {
```
Comment (AntonPozdeev) [inline src/main/java/my/pr/service/OrderService.java:131]: put it in separate class

File: src/main/java/my/pr/model/Order.java (hunk 3 above)
Comment (MM3103) [reply src/main/java/my/pr/model/Order.java]: fixed

File: src/main/java/my/pr/model/Order.java (hunk 4 above)
Comment (MM3103) [reply src/main/java/my/pr/model/Order.java]: fixed

File: src/main/java/my/pr/repository/OrderRepository.java (hunk 5 above)
Comment (MM3103) [reply src/main/java/my/pr/repository/OrderRepository.java]: fixed

File: src/main/java/my/pr/service/OrderService.java (hunk 6 above)
Comment (MM3103) [reply src/main/java/my/pr/service/OrderService.java]: fixed

**Pull request merged**